akka-docs/src/main/paradox/typed/fault-tolerance.md

```diff
@@ -38,3 +45,22 @@ Java
 For a full list of strategies see the public methods on `SupervisorStrategy`
 
 
+## Bubble failures up through the hierarchy
+
+In some scenarios it may be useful to push the decision about what to do on a failure upwards in the Actor hierarchy
+ and let the parent actor handle what should happen on failures (in untyped Akka Actors this is how it works by default).
+
+For a parent to be notified when a child is terminated it has to `watch` the child. If the child was stopped because of
+a failure this will be included in the `Terminated` signal in the `failed` field.
+
+If the parent in turn does not handle the `Terminated` message it will itself fail with an `akka.actor.typed.DeathPactException`.
+This means that a hierarchy of actors can have a child failure bubble up making each actor on the way stop but informing the
+top-most parent that there was a failure and how to deal with it, however, the original exception that caused the failure
+will only be available to the immediate parent.
```

Comment:
does that mean that if one for some reason want to bubble the original exception up another level it should be done by handling `Terminated` and re-throw the exception in the failed field?

perhaps obvious, but worth mentioning?